xclim/indicators/atmos/_precip.py

```diff
@@ -112,6 +113,16 @@ class HrPrecip(Hourly):
     compute=indices.wetdays,
 )
 
+wetdays_prop = Precip(
+    identifier="wetdays_prop",
+    units="1",
+    standard_name="proportion_of_days_with_lwe_thickness_of_precipitation_amount_at_or_above_threshold",
```

Comment:
```suggestion
```
We try to only use standard names present in the official table (http://cfconventions.org/Data/cf-standard-names/78/build/cf-standard-name-table.html).